docs/draft_production_docs/oadp-self-service.md

```diff
@@ -0,0 +1,484 @@
+| `snapshotMoveData`                         | ✅ Yes       |                          |                 |
+| `datamover`                                | ✅ Yes       |                          |                 |
+| `uploaderConfig.parallelFilesUpload`       | ✅ Yes       |                          |                 |
+| `hooks`                                    |              |                          | ⚠️ special case |
+
+An example enforcement set in the DPA spec to enforce the 
+  * ttl to be set to "158h0m0s"
+  * snapshotMoveData to be set to true
+
+```
+  nonAdmin:
+    enable: true
+    enforcedBackupSpec.ttl: "158h0m0s"
+    enforcedBackupSpec.snapshotMoveData: true
+```
+
+#### Restricted NonAdminRestore NAR
+
+NonAdminRestores spec fields can also be restricted by the cluster administrator.  The following NAR spec fields are currently supported for template enforcement:
+
+| **Field**                     | **Admin Enforceable** | **Restricted**    | **special case** |
+|-------------------------------|--------------|--------------------|-----------------|
+| `backupName`                  | ❌ No        |                    |                 |
+| `scheduleName`                | ❌ No        | ✅ Yes             |                 |
+| `itemOperationTimeout`        | ✅ Yes       |                    |                 |
+| `uploaderConfig`              | ✅ Yes       |                    |                 |
+| `includedNamespaces`          | ❌ No        | ✅ Yes             |                 |
+| `excludedNamespaces`          | ❌ No        | ✅ Yes         |                 |
+| `includedResources`           | ✅ Yes       |                |                 |
+| `excludedResources`           | ✅ Yes       |                |                 |
+| `restoreStatus`               | ✅ Yes       |                |                 |
+| `includeClusterResources`     | ✅ Yes       |                |                 |
+| `labelSelector`               | ✅ Yes       |                |                 |
+| `orLabelSelectors`            | ✅ Yes       |                |                 |
+| `namespaceMapping`            | ❌ No        | ✅ Yes         |                 |
+| `restorePVs`                  | ✅ Yes       |                |                 |
+| `preserveNodePorts`           | ✅ Yes       |                |                 |
+| `existingResourcePolicy`      |              |                | ⚠️ special case |
+| `hooks`                       |              |                | ⚠️ special case |
+
+
+
+## OpenShift Console and OADP Self-Service
+
+At the time of writing OADP self-service objects can only be created in the OpenShift Console via the API Explorer.
+
+Navigate to: Administrator -> Home -> API Explorer -> Filter on `NonAdmin`. Choose the object you wish to create.
+
+   * NonAdminBackup
+   * NonAdminRestore
+   * NonAdminBackupStorageLocation
+
+Click on instances and the create button.
+
+
```

Comment:
(just adding a comment so that we don't forget)
- NADR
- Latest CRD updates

Reply:
aye